tools/test-js-with-node

```diff
@@ -204,6 +205,7 @@ EXEMPT_FILES = make_set(
         "static/js/unread.js",
         "static/js/unread_ops.js",
         "static/js/unread_ui.js",
+        "static/js/upload.js",
```

Comment:
This commit seems to not have `git add`ed the new files?

Reply:
They're existing files. I just want to remove them from needing 100% codecov.

Reply:
Oh, I somehow misread the commit message as "Extract" not "Exclude. OK.

As a nit, I'd have done just `node:` or "node tests" as the commit message title prefix -- the part before the colon is supposed to be the part of Zulip that's being modified, and "node tests" is that part -- "node codecov" is more in the weeds.

Reply:
Updated the commit message to say that.